core/src/main/java/lucee/runtime/db/QoQ.java

```diff
@@ -396,38 +697,51 @@ private Object executeOperation(PageContext pc, SQL sql, Query qr, Operation ope
 		if (operation instanceof Operation3) {
 			Operation3 op3 = (Operation3) operation;
 			int o = op3.getOperator();
-			if (o == Operation.OPERATION3_BETWEEN) return executeBetween(pc, sql, qr, op3, row);
-			if (o == Operation.OPERATION3_LIKE) return executeLike(pc, sql, qr, op3, row);
+			if (o == Operation.OPERATION3_BETWEEN) return executeBetween(pc, sql, source, op3, row);
+			if (o == Operation.OPERATION3_LIKE) return executeLike(pc, sql, source, op3, row);
 		}
 
 		if (!(operation instanceof OperationN)) throw new DatabaseException("invalid syntax for SQL Statement", null, sql, null);
 
 		OperationN opn = (OperationN) operation;
 
-		String op = StringUtil.toLowerCase(opn.getOperator());
+		String op = opn.getOperator();
 		Expression[] operators = opn.getOperants();
 
-		/*
-		 * if(count==0 && op.equals("?")) { int pos=sql.getPosition(); if(sql.getItems().length<=pos) throw
-		 * new DatabaseException("invalid syntax for SQL Statement",null,sql); sql.setPosition(pos+1);
-		 * return sql.getItems()[pos].getValueForCF(); }
-		 */
 		// 11111111111111111111111111111111111111111111111111111
 		if (operators.length == 1) {
-			Object value = executeExp(pc, sql, qr, operators[0], row);
+
+			Object value = null;
+			Object[] aggregateValues = null;
+
+			// Aggregate operations use the entire array of values for the column instead
+			// of a single value at a given row
+			if (operation instanceof OperationAggregate) {
+				// count() has special handling below
+				if (!op.equals("count")) {
+					aggregateValues = executeAggregateExp(pc, sql, source, operators[0], true, false);
+				}
+			}
+			else {
+				value = executeExp(pc, sql, source, operators[0], row);
+			}
 
 			// Functions
 			switch (op.charAt(0)) {
 			case 'a':
 				if (op.equals("abs")) return new Double(MathUtil.abs(Caster.toDoubleValue(value)));
 				if (op.equals("acos")) return new Double(Math.acos(Caster.toDoubleValue(value)));
+				https: // ortussolutions.slack.com/archives/GUEEFNMDX
```

Comment:
that url should break the compiler ...

Reply:
@michaeloffner  I have no idea how that got there!  Must have been an accidental paste.  Removing...

Reply:
;-)